Makes events z-aware

comma opened the pull request:
No longer need hacks to stop overmap ships from triggering events. Events now affect appropriate zlevels, defaulting to station zlevels where not specified.

<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You find a README and example file in .\html\changelogs\ for further instructions.
-->

comma commented:
I didn't touch things in gamemodes/events because I am not entirely sure if we are even using those.

Chiquiii commented:
Does this fix certain events not firing?

comma commented:
No, doesn't touch event manager.

PurpleMartinJCK commented on baystation12.dme:
```diff
@@ -1382,7 +1382,6 @@
 #include "code\modules\events\brand_intelligence.dm"
 #include "code\modules\events\camera_damage.dm"
 #include "code\modules\events\carp_migration.dm"
-#include "code\modules\events\comms_blackout.dm"
```
Why remove this event?

comma replied:
Because 
1.It's unused code, not referenced by anything.
2.There's second version of that event that IS used.

PurpleMartinJCK commented on code/modules/events/communications_blackout.dm:
```diff
@@ -1,21 +1,22 @@
 /datum/event/communications_blackout/announce()
 	var/alert = pick(	"Ionospheric anomalies detected. Temporary telecommunication failure imminent. Please contact you*%fj00)`5vc-BZZT", \
-						"Ionospheric anomalies detected. Temporary telecommunication failu*3mga;b4;'1v�-BZZZT", \
+						"Ionospheric anomalies detected. Temporary telecommunication failu*3mga;b4;'1v�-BZZZT",
```
There are weird characters mixed in here that aren't displaying.

PurpleMartinJCK commented on code/modules/events/communications_blackout.dm:
```diff
@@ -1,21 +1,22 @@
-						"Ionospheri:%� MCayj^j<.3-BZZZZZZT", \
-						"#4nd%;f4y6,>�%-BZZZZZZZT")
-
+						"Ionospheri:%� MCayj^j<.3-BZZZZZZT", \
```
Characters here too.

Techhead0 commented on code/game/objects/items/devices/radio/radio.dm:
```diff
@@ -728,6 +728,7 @@
 	anchored = 1
 	simulated = 0
 	channels=list("Engineering" = 1, "Security" = 1, "Medical" = 1, "Command" = 1, "Common" = 1, "Science" = 1, "Supply" = 1, "Service" = 1, "Exploration" = 1)
+	anchored = 1
```
Double-defined. See above.

Techhead0 commented on code/modules/events/communications_blackout.dm:
```diff
@@ -1,21 +1,20 @@
-						"Ionospheri:%� MCayj^j<.3-BZZZZZZT", \
-						"#4nd%;f4y6,>�%-BZZZZZZZT")
-
+						"Ionospheri:%� MCayj^j<.3-BZZZZZZT", 						"#4nd%;f4y6,>�%-BZZZZZZZT"
```
Odd characters here and above again. Your editor really hates `£` and `¬`.

comma committed:
Makes events z-aware

No longer need hacks to stop overmap ships from triggering events. Events now affect appropriate zlevels, defaulting to station zlevels where not specified.
